var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -345,6 +345,9 @@ def nvariant_bool(feature):
             '-DBUILD_TESTING:BOOL=OFF',
             '-DOpenGL_GL_PREFERENCE:STRING=LEGACY']
 
+        if spec.satisfies('@master'):
```

Comment:
This should also apply to >= 5.11 so it dosn't need to be changed once the 5.11 release cycle starts.
```suggestion
        if '@5.11:,master' in spec:
```